modules/installation-special-config-chrony.adoc

```diff
@@ -59,12 +59,13 @@ storage:
 <1> On control plane nodes, substitute `master` for `worker` in both of these locations.
 <2> Specify an octal value mode for the `mode` field in the machine config file. After creating the file and applying the changes, the `mode` is converted to a decimal value. You can check the YAML file with the command `oc get mc <mc-name> -o yaml`.
 <3> Specify any valid, reachable time source, such as the one provided by your DHCP server.
+
 +
 [NOTE]
 ====
 For all-machine to all-machine communication, the Network Time Protocol (NTP) on UDP is port `123`. If an external NTP time server is configured, you must open UDP port `123`.
 ====
-ifndef::restricted[Alternately, you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`.]
+ifndef::restricted[Alternately, you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`. When you use NTP with your DHCP server, you must set the `sourcedir /run/chrony-dhcp` parameter in the `chrony.conf` file.]
```

Comment:
```suggestion
ifndef::restricted[Alternatively, you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`. When you use NTP with your DHCP server, you must set the `sourcedir /run/chrony-dhcp` parameter in the `chrony.conf` file.]
```
I realize this is outside the scope of your changes, but I think you mean "Alternatively" here, not "Alternately". See IBM Style word usage: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#alternate